assets/sass/04-elements/links.scss

```diff
@@ -0,0 +1,24 @@
+/*
+ * text-underline-offset doesn't work in Chrome at all 👎
+ * But looks nice in Safari/Firefox, so let's keep it and
+ * maybe Chrome will support it soon.
+ */
+a {
+	color: var(--global--color-black);
```

Comment:
If the color needs to be overwritten here I believe it still needs to be     `color: var(--wp--style--color--link, var(--global--color-primary));` and not black?
Hopefully @kjellr can confirm.

Reply:
Yes, that's right. That fallback stack should get this working nicely with the Custom Link color functionality in Gutenberg.